### DISCUSSION TITLE
Cannot get accepted to the Student program.

### DISCUSSION THREAD

**Discussion by MrTsak** [community/community#143187] [GitHub Education]
### Hi everyone,

I have tried providing every information it asks, yet I keep getting rejected no matter how many different things I try, it has gotten to a point where it asks me why I am away from my campus, well guess what, there isn't a campus and I live in the same city as my studying place, which is even recognised by github.

**Comment by CemWebDev** [comment]
Hi @MrTsak,

To help resolve your GitHub Student Pack application status, I recommend trying to link your account through the **GitHub Mobile** app. Some users have reported that after verifying their accounts on GitHub Mobile, their applications were processed more quickly. You can try downloading GitHub Mobile and linking your account from there.

Additionally, make sure to link your **student email** to your GitHub account, as this can help with the verification process.

If this doesn't resolve the issue, I suggest reaching out directly to the GitHub Education support team. They can provide more detailed information regarding the "outstanding discount request" status on your application, which might expedite the process.

Hope your application gets sorted out soon!

**Reply by MrTsak** [reply]
The issue is that we don't get a student email, I'll talk with my institute about it.
